### ISSUE TITLE
Ambiguous buttons (language Italy)

### ISSUE THREAD

**Issue by danilort** [dbeaver/dbeaver#11722]
#### System information: 
- Operating system (distribution) and version: Windows 10 versione 20H2 build 19042.804
- DBeaver version: DBeaver Community 21.0.0.202103021012 (LANGUAGE: ITALY)
- Additional extensions: NULL

#### Connection specification:
- Database name and version: MySQL 8.0.21
- Driver name: Driver for MySQL 8 and later
- Do you use tunnels or proxies (SSH, SOCKS, etc)? : NULL

#### Describe the problem you're observing:
Using the Italian language, there are two ambiguous buttons.

#### Steps to reproduce, if exist:
1) Create a script with this code:
```
CREATE TABLE `tabella1` (
    `id` int PRIMARY KEY NOT NULL AUTO_INCREMENT,
    `dataora` datetime DEFAULT NULL
);

ALTER TABLE `tabella1` COMMENT = 'Tabella di prova';
```
2) MENU: editor SQL>>esegui script SQL  (first)

3) MENU: editor SQL>>esegui script SQL   (second)

![immagine](https://user-images.githubusercontent.com/16148184/111064030-2c776780-84b2-11eb-87b3-0127bc8092f0.png)

There are two "Ignora" buttons.

**Comment by uslss** [comment]
thanks for the report

**Comment by serge-rider** [comment]
Was fixed in #11705

**Comment by uslss** [comment]
verified
